tests/FINDOLOGIC/Export/Tests/CSVSerializationTest.php

```diff
@@ -36,6 +36,8 @@ class CSVSerializationTest extends TestCase
 
     private const CSV_PATH = '/tmp/findologic.csv';
 
+    private const ALTERNATE_CSV_PATH = '/tmp/findologic-new.csv';
```

Comment:
Let's change this to how it will look in the real world.

```suggestion
    private const ALTERNATE_CSV_PATH = '/tmp/findologic.new.csv';
```

Reply:
:wrench: This comment has not been addressed.